src/server/svr_func.c

```diff
@@ -5277,6 +5317,51 @@ is_vnode_prov_done(char * vnode)
 	set_task(WORK_Immed, 0, do_provisioning, (void *) NULL);
 }
 
+/**
+ * @brief
+ *	Call check_and_run_jobs(prov_vnode_info) from a timed task to ensure that
+ *	hooks are synced with the node before job get start
+ *
+ * @param[in,out]       ptask - work task structure contains prov_vnode_info
+ *
+ * @return	void
+ *
+ */
+
+static void 
+is_sync_hooks_done(struct work_task *ptask)
+{
+	struct  pbsnode         *pnode = NULL;
+	struct prov_vnode_info * prov_vnode_info;
+
+	assert(ptask->wt_parm1 != NULL);
+
+	prov_vnode_info = (struct prov_vnode_info *) ptask->wt_parm1;
+	pnode = (struct pbsnode *) find_nodebyname(prov_vnode_info->pvnfo_vnode);
+
+	DBPRT(("%s: calling [check_and_run_jobs] from is_sync_hooks_done\n", __func__))
+
+	if (check_and_run_jobs(prov_vnode_info) == -1)
```

Comment:
Hi Riyaz,
   As discussed during our sit down review, I guess we don't need this call anymore. We can start the job from here itself if do_sync_mom_hookfiles and _sync_mom_hookfiles_running is zero.